centaur/src/main/resources/integrationTestCases/cwl/bcbio/prealign-workflow/main-prealign-samples.json

```diff
@@ -287,6 +305,28 @@
             ]
         }
     ],
+    "genome_resources__variation__gnomad_exome": [
+        {
+            "class": "File",
+            "path": "gs://bcbiodata/test_bcbio_cwl/testdata/genomes/hg19/variation/gnomad_exome.vcf.gz",
```

Comment:
These file (and the `.tbi` are not in the `bcbiodata` bucket yet. I've emailed Brad to see if he can add them there and then this should work